deps/verifier/src/snp/mod.rs

```diff
@@ -104,7 +108,8 @@ impl Verifier for Snp {
 
         verify_report_signature(&report, &cert_chain, &self.vendor_certs)?;
 
-        if report.version != 2 {
+        // See Trustee Issue#589 https://github.com/confidential-containers/trustee/issues/589
+        if report.version < REPORT_VERSION_MIN || report.version > REPORT_VERSION_MAX {
             return Err(anyhow!("Unexpected report version"));
```

Comment:
Could we also add some context including `SNP` to the error information here? Because the information
```
ERROR kbs::http::error] Attestation failed: Verifier evaluate failed: Unexpected report version
```

is somehow confusing as it could be KBS protocol's error.